articles/azure/azs-how-add-managed-disks-portal.md

```diff
@@ -96,26 +96,29 @@ If you're adding a new disk:
 
     - **Location** - This will be frn00006, which is the location of the Azure Stack Hub.
 
-    - **Account type** - Standard disks (HDD) are backed by magnetic drives and are preferable for applications where data is accessed infrequently.
-
-        > [!Note]
-        > Currently, only standard disks (HDD) are available in UKCloud for Microsoft Azure.
+    - **Account type** - Premium disks (SSD) are backed by solid state drives and offer consistent, low-latency performance. They provide the best balance between price and performance, and are ideal for I/O-intensive applications and production workloads. Standard disks (HDD) are backed by magnetic drives and are preferable for applications where data is accessed infrequently.
```

Comment:
This needs to be put back to what it was as Azure Stack Hub does not differentiate between Standard and Premium disks. Underlying infrastructure is the same, Premium disk option is only left for API compatibility so we can add that to the note but remove everything else.

Reply:
@cblackuk Updated wording in latest commit, let me know what you think.

Reply:
And again in f4a246d